### PR TITLE
DOC-1727

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+:::note
+Optionally, you can enable token authentication for the controller and registry as described in [Configuring Token Authentication](https://docs.veertu.com/anka/anka-build-cloud/advanced-security-features/token-authentication) in the Anka documentation.
+::: 
+
+
+### Install the Harness delegate and runner
+Set up the Harness delegate and runner.  
+
+You can run your delegate and runner on [Docker](define-a-docker-build-infrastructure.md), [MacOS](./define-macos-build-infra-with-anka-registry.md), [AWS](./set-up-an-aws-vm-build-infrastructure.md), [Azure](./define-a-ci-build-infrastructure-in-azure.md), and [Google Cloud Platform](./define-a-ci-build-infrastructure-in-google-cloud-platform.md) build infrastructures.
```
Comment (eoinmcafee00) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md:64]: do these links still work?

**Review by eoinmcafee00 — DISMISSED** [review]

**Comment by eoinmcafee00** [comment]
Just the one comment @douglas-j-bothwell

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md:64]: Yup, I checked them all locally and they work.

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+title: Define a MacOS Build Infrastructure and Anka Registry
+description: This topic describes how to set up a Harness MacOS build farm that uses an Anka registry and controller.
+sidebar_position: 45
+---
+
+:::note
+Currently, this feature is behind the Feature Flag `CI_VM_INFRASTRUCTURE`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+:::
+
+This topic describes the high-level workflow for setting up a Harness MacOS build farm that works with [Anka's virtualization platform for MacOS](https://docs.veertu.com/anka/what-is-anka/). Using this workflow, you can set up a highly scalable build farm with multiple VMs and nodes to run your MacOS builds with Harness CI.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md:11]:
```suggestion
This topic describes the high-level workflow for setting up a Harness macOS build farm that works with [Anka's virtualization platform for MacOS](https://docs.veertu.com/anka/what-is-anka/). Using this workflow, you can set up a highly scalable build farm with multiple VMs and nodes to run your macOS builds with Harness CI. 
```

macOS has lowercase M. Mac by itself and Mac mini are uppercase. Example: https://www.apple.com/macos/ventura/

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+:::
+
+This topic describes the high-level workflow for setting up a Harness MacOS build farm that works with [Anka's virtualization platform for MacOS](https://docs.veertu.com/anka/what-is-anka/). Using this workflow, you can set up a highly scalable build farm with multiple VMs and nodes to run your MacOS builds with Harness CI. 
+
+The following diagram shows how Harness CI and Anka work together. Once you set up the Harness and Anka components, you can easily scale up your build farm with additional templates, build nodes, and VMs. 
+
+![](./static/macos-build-infra-with-anka-registry-mult-nodes.png)
+
+
+The following steps describe the workflow.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md:18]:
```suggestion
The following steps describe the workflow:
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+* A VM template in the registry. You can use this to create VMs on other nodes as you add them to the cluster. 
+
+:::note
+Optionally, you can enable token authentication for the controller and registry as described in [Configuring Token Authentication](https://docs.veertu.com/anka/anka-build-cloud/advanced-security-features/token-authentication) in the Anka documentation.
+::: 
+
+
+### Install the Harness delegate and runner
+Set up the Harness delegate and runner.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-macos-build-infra-with-anka-registry.md:62]:
```suggestion
### Install the Harness delegate and runner

Set up the Harness delegate and runner.  
```

Not sure if it is necessary to have the space between.

**Review by aimurphy — APPROVED** [review]
Approved w some comments

**Pull request merged**